docs/modules/components/button-group/ts/views/index.tsx

```diff
@@ -1,13 +1,13 @@
-import * as React from "react";
-import { Code } from "pragmate-ui/code";
-import { ButtonGroup } from "pragmate-ui/components";
-import { Button } from "pragmate-ui/components";
-import Content from "./content.mdx";
+import * as React from 'react';
```

Comment:
Hi @Gabojcb, please, check it out the prettier config. The spaces between the curly brackets and the importments it's necessary as standard. please, review it.

Reply:
Is perfect, now i do